src/ctia/init.clj

```diff
@@ -103,16 +96,17 @@
 (defn start-ctia!
   "Does the heavy lifting for ctia.main (ie entry point that isn't a class).
   Returns the Trapperkeeper app."
```

Comment:
Added an extra arity for an explicit config, for easy test stubbing of config. This will eventually be removed.